test/bigtest/tests/create-order-test.js

```diff
@@ -33,6 +34,10 @@ describe('Create order', () => {
     expect(form.suffixSelect.value).to.be.equal('');
   });
 
+  it('has create order button', () => {
+    expect(form.createOrderButton.isButton).to.be.true;

```

Comment:
what if it will be not a button in a future? Maybe it makes sense to check `form.createOrderButton.isPresent`

Reply:
Good point. Updated